oma-pm/src/matches.rs

```diff
@@ -399,6 +413,7 @@ pub fn real_pkg(pkg: &Package) -> Option<UniquePtr<PkgIterator>> {
     unsafe { pkg.unique() }.make_safe()
 }
 
+/// return whether this package has debug version or not.
```

Comment:
Report whether a specified package provides a matching package for debug symbols.